programs/cryptid/src/state/abbreviated_instruction_data.rs

```diff
@@ -64,6 +64,10 @@ impl AbbreviatedInstructionData {
             .map(|meta| account_infos[meta.key as usize].clone())
             .collect()
     }
+
+    pub fn get_max_account_index(&self) -> u8 {
+        self.accounts.iter().map(|meta| meta.key).max().unwrap_or(0)
+    }
 }
 impl fmt::Display for AbbreviatedInstructionData {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
I know this isn't part of this change, but thought I'd ask anyway.

Is there a chance we log/leak sensitive information? Do we implement `fmt:Display` for other structs that do carry sensitive info? Or generally speaking, is everything on Solana considered public - even the state within the program?

Reply:
Everything on Solana public.